schemas/restrictions.js

```diff
@@ -29,8 +31,7 @@ export default {
           },
           duration: {
             type: 'string',
-            // eslint-disable-next-line
-            pattern: 'P(?:(\d+(?:[\.,]\d+)?W)|(\d+(?:[\.,]\d+)?Y)?(\d+(?:[\.,]\d+)?M)?(\d+(?:[\.,]\d+)?D)?(?:T(\d+(?:[\.,]\d+)?H)?(\d+(?:[\.,]\d+)?M)?(\d+(?:[\.,]\d+)?S)?)?)'
+            pattern: new RegExp(pattern).toString().replace('/', '')
```

Comment:
`pattern` is already a RegExp correct?  And is it imperative that the slash at front/back be removed?

Reply:
More to the point, I don't think it's safe to do a global replace here as a regex string might have forward slashes in the middle that should not be replaced.  If the leading/trailing slashes do need to be removed we will want to use a more targeted approach.